newsletter_automation/newsletter/routes.py

```diff
@@ -92,17 +92,14 @@ def index():
     return render_template('home.html')
 
 
-@app.route('/articles', methods=['GET', 'POST'])
-@Authentication_Required.requires_auth
-def articles():
+def add_articles():
     "This page adds articles to the database"
     try:
         addarticlesform = AddArticlesForm(request.form)
         category = AddArticlesForm(request.form)
         if request.method == 'POST':
-            if addarticlesform.submit.data:
```

Comment:
What is the impact of removing `if addarticlesform.submit.data`? You explained why you need to do it. But what is the impact on the existing web interface?

Reply:
The impact of removing `if addarticlesform.submit.data` on the existing web interface would be, URL's would not be validated(if given URL is valid URL or not) from wtforms _form.py_ ` url = URLField('url', validators=[DataRequired(), url()])
`